torch/csrc/autograd/variable.cpp

```diff
@@ -363,6 +364,16 @@ void bump_version(const Variable& self) {
   self.unsafeGetTensorImpl()->bump_version();
 }
 
+void forbid_in_autograd(const Variable& self) {
+  TORCH_CHECK(
+      self.defined(), "cannot call forbid_in_autograd() on undefined tensor");
+  auto new_grad_fn = std::shared_ptr<torch::autograd::Error>(
+      new torch::autograd::Error(
+          "Cannot backprop through Error node, file a bug in PyTorch"),
```

Comment:
Cannot backprop through a Tensor that was marked as forbidden in backward.
Or something similar